controllers/ibmpowervsmachine_controller_test.go

```diff
@@ -24,7 +24,6 @@ import (
 
 	"github.com/IBM-Cloud/power-go-client/power/models"
 	"github.com/golang/mock/gomock"
-
```

Comment:
can we keep them in a separate sections?

Reply:
yes, changed it.